extensions/sql-database-projects/resources/templates/newSdkSqlProjectTemplate.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build">
-  <Sdk Name="Microsoft.Build.Sql" Version="0.2.0-preview" />
+  <Sdk Name="Microsoft.Build.Sql" Version="0.2.5-preview" />
```

Comment:
reminder to also update the default version in the SqlProjects repo. I think that's what's used for creating new projects. I can't remember if this template still gets used by anything.

Reply:
Zi and I just had the same convo.  I'm not seeing any references to this file, so I probably just forgot to tear it out after the SqlProjects swapover was complete.

Reply:
We can use #25797 to track the change to use SqlProjects, I can give it a go when I find some capacity